scripts/load-field-functions

```diff
@@ -0,0 +1,8 @@
+Airports:
+    LOAD
+      rowID As id,
+      REPLACE(Airport, 'Airport', '') as Airport,
```

Comment:
I don't quite understand what this does. Possibly add an inline comment on what is actually achieved?